docs/design/trace-comparison-2061.md

```
@@ -0,0 +1,138 @@
+# Trace Comparison: Expected vs Actual (#2061)
+
+With `--jobs 1 -v debug`. Vocabulary from flow-vocabulary.md.
+With right-lt fix applied (add only receives on strictly-less-than, not equal).
+
+## Functions
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)